routes/features.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+|--------------------------------------------------------------------------
+| Feature Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register all of the routes for an application.
+| It is a breeze. Simply tell Lumen the URIs it should respond to
+| and give it the Closure to call when that URI is requested.
+|
+*/
+
+/**
+ * API version number
+ */
+$apiVersion = '3';
+$apiBase = 'api/v' . $apiVersion;
+
+$router->group([
+	'prefix' => $apiBase,
+	'namespace' => 'API'
+], function () use ($router) {
+
+	// HXL
+	$router->group([
+		'prefix' => 'hxl',
+		'middleware' => ['feature:hxl']
+	], function () use ($router) {
+		$router->get('/', "HXLController@index");
+	});
+
+	// User Settings
```

Comment:
I kind prefer keeping these within the rest of the routes, so we don't have to guess which file something is in depending on if it has a feature toggle or not. I think in general feature flagged code shouldn't move because it has a feature flag or not. (maybe this is different for permanent feature flags though? ie. thing that will never be generally available?)

I do agree that `routes/web.php` is too hard to scan though. Would it work to just split it into groups ie. routes/user, routes/form, routes/post etc? (that should happen on its on PR though)